Client/Frontend/Browser/Browser.swift

```diff
@@ -97,4 +97,32 @@ class Browser: NSObject, WKScriptMessageHandler {
     func getHelper(#name: String) -> BrowserHelper? {
         return helpers[name]
     }
-}
\ No newline at end of file
+}
+
+extension WKWebView {
+
+    func runScript(src: String, js: String, callback: (AnyObject?) -> Void) {
```

Comment:
How about `runScriptFunction(function: String, fromSource source: String, callback...)` or something similar that makes it more clear what this does.